accounts/views.py

```diff
@@ -5,9 +5,12 @@
 from .models import CustomUser
 from django.views import View
 from django.core.exceptions import ObjectDoesNotExist
+from django.core.cache import cache
 from .jwt import generate_access_token, generate_refresh_token, decode_token, get_token_exp, getformat_str_token_exp, save_refresh_token
 import requests
 
+refreshToken = 'refreshToken'
```

Comment:
```suggestion
REFRESH_TOKEN = 'refreshToken'
```
파이썬에서 constant 네이밍 컨벤션은 위와 같은 스크리밍 스네이크 케이스인 것으로 알고 있어요

Reply:
처음에는 get 함수의 인자값으로 언더바를 사용하였는데, 값이 불러와지지 않아서 카멜케이스로 작성했습니다.
const 명은 스네이크 케이스로 변경해야될것 같네요.

Reply:
https://taxijjang.tistory.com/80

참고하시면 좋을것 같아요!

Reply:
[1fc7fd1](https://github.com/Almaeng-i/GuHwangJagMul-Server/pull/26/commits/1fc7fd11c95771e269c626978fb42d3c3387b526)

수정하였습니다.